quickfixj-core/src/main/java/quickfix/DefaultSessionSchedule.java

```diff
@@ -41,10 +41,17 @@
     private final int[] weekdayOffsets;
     protected static final Logger LOG = LoggerFactory.getLogger(DefaultSessionSchedule.class);
 
+    //Cache recent time data to reduce creation of calendar objects
+    private ThreadLocal<Calendar> threadLocalCalendar;
+    private ThreadLocal<TimeInterval> threadLocalRecentTimeInterval;
```

Comment:
```suggestion
    private final ThreadLocal<Calendar> threadLocalCalendar;
    private final ThreadLocal<TimeInterval> threadLocalRecentTimeInterval;
```
Could we add a `final` here to a) make it consistent with the rest of the fields and b) give a hint to the reader that these fields are initialized in the constructor?